Uppsala/UppEt3.xml

```diff
@@ -0,0 +1,557 @@
+                                    </dimensions>
+                                    <note>Data on dimensions have been adopted from the entry on the manuscript in
+                                        <bibl><ptr target="bm:alvin"/></bibl>.
+                                    </note>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">31</citedRange>
+                                </bibl> calls it a "large and even archaic script" 
+                                    (<foreign xml:lang="de">Große and gleichmäßige archaische Schrift</foreign>).</desc>
+                                <date notBefore="1400" notAfter="1499">15th century.</date>
+                            </handNote>
+                        </handDesc>
+                        
+                        <decoDesc>
+                            
+                            <decoNote xml:id="d1" type="Headband">
+                                <locus target="1r"/>
+                                <desc>Ornamental band (braided pattern) in black and red on top of the text.</desc>
+                            </decoNote>
+                            
+                        </decoDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="55r"/>
+                                    <desc type="Unclear">Under the supplication to <ref target="#ms_i1"/>, between the
+                                        <foreign xml:lang="la">cruces ansatae</foreign>, the remnants of an erased
+                                        note are visible. Only one word has been left untouched. The note is not mentioned by 
+                                        <bibl><ptr target="bm:Zettersteen1899Upsala"/></bibl> or
+                                        <bibl><ptr target="bm:Loefgren1974Katalog"/></bibl>.</desc>
+                                    <q xml:lang="gez">እግዚአብሔር፡<del rend="erasure" unit="chars" quantity="15"/></q>
+                                </item>
+                                
+                                <item xml:id="a2">
+                                    <locus target="55v"/>
+                                    <desc type="Supplication">Supplication by a later hand. The note is mentioned by 
+                                        <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">28</citedRange></bibl> and
+                                        <bibl><ptr target="bm:Zettersteen1899Upsala"/><citedRange unit="page">510</citedRange></bibl>.</desc>
```

Comment:
You do not need to refer to both catalogues in this case (I would drop it in a few other cases, whenever it does not bring anything new in comparison to Löfgren), even though the older should be in the bibliography.